src/ServicePulse.Host/vue/src/components/failedmessages/EditMessageHeader.vue

```diff
@@ -49,10 +49,10 @@ onMounted(() => {
     <input :class="{ 'header-removed': header.isMarkedAsRemoved }" class="form-control" :disabled="header.isLocked" v-model="header.value" />
   </td>
   <td>
-    <a v-if="!header.isLocked && !header.isMarkedAsRemoved" href="#" @click="markHeaderAsRemoved()">
+    <a v-if="!header.isLocked && !header.isMarkedAsRemoved" @click="markHeaderAsRemoved()">
```

Comment:
the general guidance on template generation is to not use anchors unless you're actually linking to something. These should be buttons, not anchors